config/routes.rb

```diff
@@ -331,6 +331,7 @@
         member do
           post 'template_selected'
           post 'cluster_selected'
+          post 'resource_pools'
```

Comment:
I realize the other AJAX actions are `post` too, but for an action that is purely retrieving data (the resource pools), I don't see why not using GET with params?